WordPress/openverse
# fix: broken pipenv install link in general quickstart guide

## Fixes



Fixes #3630 by @rwidom

## Description
This pull request corrects a broken link in the pipenv section of the [general setup guide](https://docs.openverse.org/general/general_setup.html#pipenv) documentation, which was leading to a 404 error:

![3630](https://github.com/WordPress/openverse/assets/127815146/9a9c0672-dcde-4fc8-87f6-f061398ac9ce)
![Screenshot 2024-01-13 at 4 32 39 AM](https://github.com/WordPress/openverse/assets/127815146/c2c6bade-78da-418a-8ad8-3313cd214e46)


The incorrect URL, `https://pipenv.pypa.io/en/latest/installation/#installing-pipenv`, has been updated to the correct URL, `https://pipenv.pypa.io/en/latest/installation.html#installing-pipenv`, ensuring users are directed to the proper installation instructions for pipenv in the documentation.


## Testing Instructions
To verify the fix:
1. Navigate to the pipenv section of the [general setup guide](https://docs.openverse.org/general/general_setup.html#pipenv) in the documentation.
2. Click on the updated link to the pipenv installation instructions.
3. Confirm that the link no longer leads to a 404 error and instead directs to the correct pipenv installation guide.


## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
